model/riscv_sys_regs.sail

```diff
@@ -150,6 +150,9 @@ val sys_enable_fdext = {c: "sys_enable_fdext", ocaml: "Platform.enable_fdext", _
 val sys_enable_zfinx = {c: "sys_enable_zfinx", ocaml: "Platform.enable_zfinx", _: "sys_enable_zfinx"} : unit -> bool
 /* whether the N extension was enabled at boot */
 val sys_enable_next = {c: "sys_enable_next", ocaml: "Platform.enable_next", _: "sys_enable_next"} : unit -> bool
+/* Whether FIOM bit of menvcfg/senvcfg is implemented. It must be if supervisor mode
+   is implemented or non-bare addressing modes are supported. */
```

Comment:
This comment is wrong - should be "It must be supported if supervisor mode is implemented *AND* non-bare addressing modes are supported." I'll update it.